FAIL: test_KeepArtifacts (test2.TestToolkitMethodsNew)

See this sometimes.
```
FAIL: test_KeepArtifacts (test2.TestToolkitMethodsNew)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/streamsadmin/topology/dev/streamsx.topology/test/python/topology/test2.py", line 91, in test_KeepArtifacts
    verifyArtifacts(self)
  File "/home/streamsadmin/topology/dev/streamsx.topology/test/python/topology/test2.py", line 47, in verifyArtifacts
    assertArchivePath(test, test.result)
  File "/home/streamsadmin/topology/dev/streamsx.topology/test/python/topology/test2.py", line 37, in assertArchivePath
    test.assertIn('archivePath', submissionResult)
AssertionError: 'archivePath' not found in <streamsx.topology.context.SubmissionResult object at 0x7fd2c682dcc0>
```

--- a/test/python/topology/test2.py
+++ b/test/python/topology/test2.py
@@ -41,14 +41,14 @@ def verifyArtifacts(test):
     if test.test_config.get('topology.keepArtifacts', False):
         # KeepArtifacts is True
         assertToolkitRoot(test, test.result)
-        if (test.test_config.get('topology.forceRemoteBuild', False) or
-                    'STREAMS_INSTALL' not in os.environ or
-                    'BUILD_ARCHIVE' == test.test_ctxtype):
-            assertArchivePath(test, test.result)
-            test.assertNotIn('bundlePath', test.result)
-        elif 'TOOLKIT' == test.test_ctxtype:
+        if 'TOOLKIT' == test.test_ctxtype:
             test.assertNotIn('bundlePath', test.result)
             test.assertNotIn('archivePath', test.result)
+        elif (test.test_config.get('topology.forceRemoteBuild', False) or
+              'STREAMS_INSTALL' not in os.environ or
+              'BUILD_ARCHIVE' == test.test_ctxtype):
+            assertArchivePath(test, test.result)
+            test.assertNotIn('bundlePath', test.result)
         else:
             assertBundlePath(test, test.result)
             test.assertNotIn('archivePath', test.result)

